Drop-In/src/main/java/com/braintreepayments/api/dropin/DropInRequest.java

```diff
@@ -70,8 +70,8 @@ public DropInRequest tokenizationKey(String tokenizationKey) {
      *
      * This method is optional. Amount is only used for 3D Secure verifications.
      *
-     * Amount must be a non-negative number, may optionally contain exactly 2 decimal places
-     * separated by '.', optional thousands separator ',', limited to 7 digits before the decimal point.
+     * This value must be a non-negative number and must match the currency format of the merchant account.
+     * It can only contain numbers and can optionally include one decimal point with exactly 2 decimal place precision (ex: x.xx).
```

Comment:
```suggestion
     * It can only contain numbers and optionally one decimal point with exactly 2 decimal place precision (e.g., x.xx).
```
Small suggestion 🙂